bindings/matrix-sdk-ffi/src/session_verification.rs

```diff
@@ -16,12 +16,17 @@ use crate::error::ClientError;
 
 #[derive(uniffi::Object)]
 pub struct SessionVerificationEmoji {
+    code: i32,
     symbol: String,
     description: String,
 }
 
 #[uniffi::export]
 impl SessionVerificationEmoji {
+    pub fn code(&self) -> i32 {
+        self.code
```

Comment:
Not sure if I need to clone here

Reply:
You don't need to clone it because it can be copied. Rust makes the difference between a copy and a clone (basically, `memcpy` vs. more complex code to create a clone of a value).